app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -331,4 +357,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void spokenText(String searchInput) {
+        if (searchInput != null) {
```

Comment:
The module should never pass back a null or empty string, because in that case, discernible text wasn't really spoken.